src/client/profile-provider.ts

```diff
@@ -70,7 +69,7 @@ export class BoundProfileProvider {
     private readonly mapAst: MapDocumentNode,
     private readonly providerName: string,
     private readonly configuration: {
-      baseUrl?: string;
+      baseUrl: string;
```

Comment:
I am not sure about the `baseUrl` change. We are counting on `serviceId.baseUrl` with fallback to `defaultService.baseUrl` now?

Reply:
Yeah, `baseUrl` must be available and if it isn't then `bind` throws. I think we don't want to support absolute uris in maps for transparency sake.